modules/core/src/webgl/classes/renderbuffer.js

```diff
@@ -29,13 +30,20 @@ export default class Renderbuffer extends Resource {
 
   constructor(gl, opts = {}) {
     super(gl, opts);
+    this.renderbufferMemoryStats = statsManager.get('Memory Usage').get('Renderbuffer Memory');
```

Comment:
Is this lookup an cheap operation? If so, maybe not save this member on the object?